Firmware/cardreader.cpp

```diff
@@ -246,7 +246,7 @@ void CardReader::startFileprint()
   {
     sdprinting = true;
     #ifdef SDCARD_SORT_ALPHA
-	  flush_presort();
+	 // flush_presort();
```

Comment:
Why comment out a single line of code under a conditional when removing the entire block would be clearer?